### PR TITLE
Update and rename momotek_abs.mtl to iso1043_abs.mtl

### PR DESCRIPTION
**Pull request by Mick3Lozzo**
Renaming the mtl file and adding tensile ultimate stress too

### CODE REVIEW

File: config/materials/iso1043_abs.mtl [hunk 1]
```diff
@@ -31,7 +31,7 @@ PARAMETERS =
   Name = PTC_MATERIAL_DESCRIPTION
   Type = String
   Default = 'ABS for CNC machining
-Source: datasheet from supplier '
+Source: datasheet from supplier'
```
Comment (Lawproto) [inline config/materials/iso1043_abs.mtl:34]: @Mick3Lozzo I would just add "Momotek" before supplier

File: config/materials/iso1043_abs.mtl [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿ND_RelParSet_K01 = {
 
-Name = MOMOTEK_ABS
+Name = iso1043_ABS
```
Comment (fiorisi) [inline config/materials/iso1043_abs.mtl:3]: @Mick3Lozzo the name can be written with capital letters, i.e. `ISO1043_ABS`.

**Comment by fiorisi** [comment]
The other parameters (PTC_YOUNG_MODULUS, PTC_POISSON_RATIO) are taken from the datasheet as well?

**Comment by Mick3Lozzo** [comment]
> The other parameters (PTC_YOUNG_MODULUS, PTC_POISSON_RATIO) are taken from the datasheet as well?

The Young Modulus, yes.
Poisson ratio was not present in the datasheet, anyway I searched for the web and for CNC ABS is around 0.35.

[Here](https://www.matweb.com/search/DataSheet.aspx?MatGUID=d16a64a389df424f8e53ed481e77477b&ckck=1) Matweb says that a CNC ABS alloy has a Poisson ratio of 0.340 - 0.380, Average value: 0.353. Some others [molded ABS](https://www.matweb.com/search/datasheet_print.aspx?matguid=eb7a78f5948d481c9493a67f0d089646&n=1) have a Poissons ration of 0.360 - 0.380, Average value: 0.364.

@fiorisi @Lawproto

File: config/materials/iso1043_abs.mtl (hunk 1 above)
Comment (Mick3Lozzo) [reply config/materials/iso1043_abs.mtl:34]: > @Mick3Lozzo I would just add "Momotek" before supplier

I don't know if we have enough digits. I'll try.

**Comment by Lawproto** [comment]
Yeah, Poisson's ratio can be a pretty rare beast to find. For me, the values you input are fine.

**Review by fiorisi — APPROVED** [review]

**Pull request merged**